RadialMenuControl/UserControl/ListSubMenu.xaml.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace RadialMenuControl.UserControl
+{
+    using RadialMenuControl.UserControl;
+    using RadialMenuControl.Components;
+    using System;
+    using System.Collections.Generic;
+    using System.Collections.ObjectModel;
+    using System.ComponentModel;
+    using System.Diagnostics;
+    using System.Runtime.CompilerServices;
+    using Windows.UI;
+    using Windows.UI.Xaml;
+    using Windows.UI.Xaml.Controls;
+    using Windows.UI.Xaml.Media;
+    using Windows.UI.Xaml.Automation.Peers;
+    using Windows.UI.Xaml.Automation.Provider;
+
+    public sealed partial class ListSubMenu : MenuBase
+    {
+        public static readonly DependencyProperty ListHeightProperty =
+            DependencyProperty.Register("ListHeightProperty", typeof(double), typeof(ListSubMenu), null);
+
+        public double ListHeight

```

Comment:
We definitely can set this Height property to diameter by default. We should however expose this as a property so that users can set it to whatever they want. What do you think?